backend/src/main/java/clf/integra/backend/controller/UserController.java

```diff
@@ -60,4 +62,10 @@ public ResponseEntity<Double> getUserBalanceById(@PathVariable UUID id) {
         }
         return ResponseEntity.ok(balance);
     }
-}
+
+    @PostMapping("/accounts/transfer")
```

Comment:
Please rename it to user/transfer, as we transfer money from one user to the other for now